src/mds/MDCache.cc

```diff
@@ -416,13 +417,12 @@ void MDCache::create_empty_hierarchy(MDSGather *gather)
   adjust_subtree_auth(rootdir, mds->get_nodeid());   
   rootdir->dir_rep = CDir::REP_ALL;   //NONE;
 
-  rootdir->fnode.accounted_fragstat = rootdir->fnode.fragstat;
-  rootdir->fnode.accounted_rstat = rootdir->fnode.rstat;
-
-  root->inode.dirstat = rootdir->fnode.fragstat;
-  root->inode.rstat = rootdir->fnode.rstat;
-  ++root->inode.rstat.rsubdirs;
```

Comment:
I think the bug is caused by removing this line

Reply:
I've fixed the issue I found here @ukernel . See the comment in the code.

Reply:
I don't see any difference in wip-pdonnell-testing-20181005.225845

Reply:
I'm saying the commits you reviewed already had been amended to fix the issue. I haven't see the failure in testing since.